apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -174,6 +182,31 @@ ConversationView.prototype = {
     );
   },
 
+  enterEditMode: function() {
+    this.showOptions();
+    this.menuAccessors.selectAppMenuOption('Select Messages');
+  },
+
+  exitEditMode: function() {
+    this.back();
+  },
+
+  selectMessage: function(messageId) {
```

Comment:
__nit:__ sorry I just realized that it can be useful in other scenarios too + it deselects as well, so better have more generic name maybe just ```tapOnMessage```?